### PR TITLE
Refine `database-backup` icon

### PR DESCRIPTION
**Pull request by danielbayley**

### CODE REVIEW

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

File: icons/database-backup.svg [hunk 1]
```diff
@@ -10,9 +10,10 @@
...
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
-  <path d="M12 12v4h4" />
-  <path d="M3 5v14c0 1.43 2.97 2.63 7 2.93" />
+  <path d="M7 14.5C4.6 14 3 13 3 12" />
+  <path d="M9.5 21.9C5.7 21.5 3 20.4 3 19" />
+  <path d="M3 19V5" />
+  <path d="M21 5v4" />
+  <path d="m11 16 2.5-2.5c1-1 2.3-1.5 3.5-1.5 2.8 0 5 2.2 5 5s-2.2 5-5 5c-1.6 0-3.1-.8-4-2" />
+  <path d="M11 12v4h4" />
```
Comment (karsa-mistmere) [inline icons/database-backup.svg:18]: One more thing: can you merge back the two paths that became separate? (the left side and the bottom)

File: icons/database-backup.svg [hunk 2]
```diff
@@ -10,9 +10,9 @@
...
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 12c0 1.18 2.03 2.2 5 2.7" />
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
-  <path d="M12 12v4h4" />
-  <path d="M3 5v14c0 1.43 2.97 2.63 7 2.93" />
+  <path d="M7 14.5C4.6 14 3 13 3 12" />
+  <path d="M9.5 21.9C5.8 21.6 3 20.4 3 19V5" />
+  <path d="M21 5v4" />
+  <path d="m11 16 2.5-2.5c1-1 2.3-1.5 3.5-1.5 2.8 0 5 2.2 5 5s-2.2 5-5 5c-1.6 0-3.1-.8-4-2" />
```
Comment (karsa-mistmere) [inline icons/database-backup.svg:16]: Hm. Maybe this have a bit better curvature?
```suggestion
  <path d="M13 20c.9 1.2 2.4 2 4 2 2.8 0 5-2.2 5-5s-2.2-5-5-5c-2.37 0-4.45 1.59-6 4" />
```
image
The difference is quite subtle on 100%, but it's there:
image

Comment (danielbayley) [reply icons/database-backup.svg:16]: Well this is how wonky it currently looks, so either way is better than this…

Untitled

I personally prefer the more geometric version, which also seems to match the existing `refresh` and other icons… Either way, they should all be consistent.

Comment (danielbayley) [reply icons/database-backup.svg:16]: > Maybe this have a bit better curvature?

So other arrows in this style (see https://lucide.dev/icons?search=ccw) don’t have this proposed extra curvature… I was going for consistency with the existing version… I think either we have to change them all, or stick with the existing approach… Thoughts? @karsa-mistmere @ericfennis @jguddas

Comment (ericfennis) [reply icons/database-backup.svg:16]: @danielbayley @karsa-mistmere I designed these. That there is no curvature on the end is on purpose. 
I can give you an example with the current refresh icons, these also haven't curved arrow ends (which I designed as well). To be short on this answer: It looks better on 100% scale. Which is more important than that the icon will look good when zoomed in.

Here is a visual comparison that explains it.
image

Comment (danielbayley) [reply icons/database-backup.svg:16]: @ericfennis Yeah I thought the same, which is why I matched this to the existing ones. Should be good to go then…? @karsa-mistmere

Comment (karsa-mistmere) [reply icons/database-backup.svg:16]: I did provide a 100% view, the left one is the current, the right one is with the proposed geometric curve:
image

To me, the geometric curve looks cleaner at 100%. 😅

Comment (karsa-mistmere) [reply icons/database-backup.svg:16]: @ericfennis: This isn't perfect, but it's very very close to the current icon but has a less wonky path :
database-backup

image
image

Comment (danielbayley) [reply icons/database-backup.svg]: @karsa-mistmere @ericfennis ccw arrow now perfectly matches #1053…

Comment (danielbayley) [reply icons/database-backup.svg]: @karsa-mistmere Dirty trick, but it kinda works… 😅

Untitled

Comment (ericfennis) [reply icons/database-backup.svg]: @danielbayley awesome!

**Review by ericfennis — APPROVED** [review]
I think this one is good to go!

File: icons/database-backup.svg [hunk 3]
```diff
@@ -10,9 +10,9 @@
   stroke-linejoin="round"
 >
   <ellipse cx="12" cy="5" rx="9" ry="3" />
-  <path d="M3 12c0 1.18 2.03 2.2 5 2.7" />
-  <path d="M21 5v4.5" />
-  <path d="m12 16 1.27-1.35a4.75 4.75 0 1 1 .41 5.74" />
+  <path d="M3 5v14c0 1.4 3 2.7 7 3" />
+  <path d="M3 12c0 1.2 2 2.5 5 3" />
+  <path d="M21 5v4" />
+  <path d="M13 20c.9 1.2 2.3 2 4 2 2.8 0 5-2.2 5-5 0-1.2-.4-2.3-1.3-3.2-1.8-1.8-4.6-1.8-6.4 0L12 16" />
```
Comment (karsa-mistmere) [inline icons/database-backup.svg:16]: A very slight optimization:
```suggestion
  <path d="M13 20a5 5 0 0 0 9-3 4.5 4.5 0 0 0-4.5-4.5c-1.33 0-2.54.54-3.41 1.41L12 16" />
```

Comment (danielbayley) [reply icons/database-backup.svg:16]: This results in a shape slightly off, at 10x10.0003px… Sure?

Comment (karsa-mistmere) [reply icons/database-backup.svg:16]: Well _without_ the arrow head, yours is 10x9.55 in Figma, my proposal is 10x9.5, hence the optimization. 😄

Comment (danielbayley) [reply icons/database-backup.svg:16]: > Well _without_ the arrow head, yours is 10x9.55 in Figma, my proposal is 10x9.5, hence the optimization. 😄

I think this is another Figma vs Illustrator thing… It was 10x9.5 in Illustrator 🙈

**Pull request merged**